test/sign.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { exec } from "child_process";
+var assert = require("assert");
+const testData =
```

Comment:
Adding comments on what those variable contain and how they were generated would be helpful to maintain them.